Makefile

```diff
@@ -77,5 +78,9 @@ generate-website-docs:
 generate-artifacthub-artifacts:
 	cd $(ARTIFACTHUB_SCRIPT_DIR); go test -v && go run hub.go
 
+.PHONY: validate
+validate:
+	cd $(VALIDATE_SCRIPT_DIR); go test -v & go run validate.go
```

Comment:
```suggestion
	cd $(VALIDATE_SCRIPT_DIR); go test -v && go run validate.go
```

Reply:
this is kinda confusing, since unit tests will pass but "go run" will panic

Reply:
It's possible, but I prefer to run unit tests before proceeding with the validation code to ensure it's fully tested beforehand, especially since we're not building anything.

Is there a better way to test it before `run`ning it?

Reply:
unit tests can be in a separate/parallel workflow then execution so we don't have wait for them.

Reply:
can we separate out a "test" make target and run that in CI?

Reply:
makes sense. I added a separate stage for testing scripts.